doc/sphinxext/docscrape.py

```diff
@@ -8,6 +8,11 @@
 import collections
 import sys
 
+if sys.version_info >= (3, 10):
```

Comment:
This foes not always work well, specially with dev version  (like `3.10.1dev0`).

Ca you use packaging like we do [here](https://github.com/dipy/dipy/blob/89e4ab6a4a2c40f8681e02f930a3fc2371f51f82/dipy/conftest.py#L15)

Reply:
This foes not always work well, specially with dev version (like 3.10.1dev0).
What do you mean by foes ?

Reply:
sorry, typo. 

it is `does` and not `foes`

Reply:
Is this fine ? 
```
from packaging.version import Version
if Version(sys.__version__) >= Version('3.10'):
    from collections.abc import Mapping 
else:
    from collections import Mapping 
```

Reply:
yes